src/components/cocktail.js

```diff
@@ -21,10 +22,23 @@ export default class Cocktail extends React.Component {
     this.setState({loc: this.loc.current.value})
   }
 
+  handleFindNewDrink = () => {
+    console.log('dispatch get another drink');
```

Comment:
⛏ Looks like there are some leftovers in here, as well. See also below on lines 37-38